app/code/Magento/Downloadable/view/adminhtml/templates/product/composite/fieldset/downloadable.phtml

```diff
@@ -6,37 +6,42 @@
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 // @deprecated
 ?>
-<?php /* @var $block \Magento\Downloadable\Block\Adminhtml\Catalog\Product\Composite\Fieldset\Downloadable */ ?>
+<?php
```

Comment:
Please remove not needed closing and opening PHP tags

Reply:
fixed